app/controllers/v1/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+class V1::BaseController < ApplicationController
```

Comment:
It's just a question, you already have my :+1: 

Why you create `BaseController` and not use `ApplicationController` instead?